src/styles/_mixins.scss

```diff
@@ -6,4 +6,33 @@
 
   @mixin expand-transition {
     transition: width 0.3s ease-in-out;
-  }
\ No newline at end of file
+  }
+
+  @mixin base-input {
```

Comment:
I very much like the idea of reusing components instead of reusing styles - it's easier to manage, easier to test and easier introduce changes. Maybe it is possible to move this style to the component and reuse the component?

Reply:
That solution was forced by inability to utilize base input inside password input component so i had to share those styles but now its redone and problem automatically disappears 😎